src/App.js

```diff
@@ -1,36 +1,82 @@
 import React, { Component } from 'react';
-import movieData from './movieData.js';
 import MoviesContainer from './MoviesContainer.js';
 import MovieDetails from './MovieDetails.js'
-import dummyData from './dummyData.js'
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state ={
-      movies: movieData.movies,
+      movies: [],
       currentMovie: '',
       moviesContainer: true,
-      dummyData: dummyData
+      error: false
     }
   }
 
+  componentDidMount = () => {
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies`)
+      .then(response => {
+        if (response.status >= 200 && response.status <= 299) {
```

Comment:
This is great. You can also use the `response.ok` property to check this same range of status code values.